src/logger.rs

```diff
@@ -39,9 +39,3 @@ const ERROR: u32 = 1;
 const WARNING: u32 = 2;
 const INFO: u32 = 3;
 const DEBUG: u32 = 4;
-
-#[link(wasm_import_module = "index")]
-extern "C" {
-    #[link_name = "log.log"]
-    fn log(level: u32, message: &AscStr);
```

Comment:
This was just from an example?

Reply:
Sorry, just say that you moved it to `host.rs`

Reply:
Yeah, the idea here is we have:
- `host.rs`: Contains all the `extern "C"` blocks for importing the functions made available by the Wasm host
- `mock.rs`: Contains mocked versions of the host functions so that subgraph crates (like the `deep-thought` example) can just run `cargo test` and have it just work (without having to have an external "mock host" implementation that would load up the subgraph, start the test harness, and run some tests).

Which one is used is determined by conditional compilation based on the compilation target (i.e. whether its being built for Wasm or not).